sqldelight-idea-plugin/src/main/kotlin/app/cash/sqldelight/intellij/intentions/AddImportIntention.kt

```diff
@@ -104,4 +120,36 @@ internal class AddImportIntention(
       replaceString(0, endOffset, newImports.sorted().joinToString("\n"))
     }
   }
+
+  override fun showHint(editor: Editor): Boolean {
+    if (element.reference?.resolve() != null) {
+      return false
+    }
+
+    val sqlDelightFile = element.containingFile as SqlDelightFile
+    val stmtList = PsiTreeUtil.findChildOfType(sqlDelightFile, SqlDelightImportStmtList::class.java)
+    val importStmtList = stmtList?.importStmtList.orEmpty()
+    for (stmt in importStmtList) {
+      for (cls in classes) {
+        if (stmt.javaType.textMatches(cls.qualifiedName.orEmpty())) {
+          return false
+        }
+      }
+    }
+    val name = classes[0].name ?: return false
```

Comment:
I believe this is the behavior in Java/Kotlin
```suggestion
    val name = classes[0].qualifiedName ?: return false
```

Reply:
Yes, thanks!